.circleci/config.yml

```diff
@@ -201,6 +213,28 @@ jobs:
           name: backend
           command: make -C src/backend test
 
+  coverage:
+    docker:
+      - <<: *frontend_base
+
+    steps:
+      - *restore_repo
+      - run: *install_dependencies
```

Comment:
:thought_balloon: nothing in the rake tasks needs the rails app right? This could be outside `lib/`...

Reply:
Well, it uses Rails.root to find the files. What do you gain by not having it in lib?

Reply:
You don't have to install dependencies and can use a smaller container...